examples/STM32-LWIP-WOLFSSL/Src/sentry.c

```diff
@@ -0,0 +1,387 @@
+{
+    const struct wolfsentry_event *parent_event = wolfsentry_route_parent_event(route);
+    (void)wolfsentry;
+    (void)handler_arg;
+    (void)route_table;
+    (void)action_results;
+
+    /*printf("action callback: a=\"%s\" parent_event=\"%s\" trigger=\"%s\" t=%u r_id=%u caller_arg=%p\n",
+           wolfsentry_action_get_label(action),
+           wolfsentry_event_get_label(parent_event),
+           wolfsentry_event_get_label(trigger_event),
+           action_type,
+           wolfsentry_get_object_id(route),
+           caller_arg);*/
+    return 0;
+}
+
+/* Initialize wolfSentry with a config file */
+int sentry_init()
+{
+    wolfsentry_errcode_t ret;
+    wolfsentry_ent_id_t id;
+
+    struct wolfsentry_eventconfig config = { .route_private_data_size = 32, .route_private_data_alignment = 16 };
+    ret =  wolfsentry_init(NULL /* hpi */, &config,
+                           &wolfsentry);
+
+    if (ret < 0)
+    {
+        printf("unable to initialize wolfSentry, code: %ld\n", WOLFSENTRY_ERROR_DECODE_ERROR_CODE(ret));
+    }
+
+    /* Insert the possible actions into wolfSentry */
+    wolfsentry_action_insert(      wolfsentry,
+                                   "handle-insert",
+                                   WOLFSENTRY_LENGTH_NULL_TERMINATED,
+                                   WOLFSENTRY_ACTION_FLAG_NONE,
+                                   test_action,
+                                   NULL,
+                                   &id);
+
+    wolfsentry_action_insert(
+        wolfsentry,
+        "handle-delete",
+        WOLFSENTRY_LENGTH_NULL_TERMINATED,
+        WOLFSENTRY_ACTION_FLAG_NONE,
+        test_action,
+        NULL,
+        &id);
+
+    wolfsentry_action_insert(
+        wolfsentry,
+        "handle-match",
+        WOLFSENTRY_LENGTH_NULL_TERMINATED,
+        WOLFSENTRY_ACTION_FLAG_NONE,
+        test_action,
+        NULL,
+        &id);
+
+    wolfsentry_action_insert(
+        wolfsentry,
+        "notify-on-match",
+        WOLFSENTRY_LENGTH_NULL_TERMINATED,
+        WOLFSENTRY_ACTION_FLAG_NONE,
+        test_action,
+        NULL,
+        &id);
+
+    wolfsentry_action_insert(
+        wolfsentry,
+        "handle-connect",
+        WOLFSENTRY_LENGTH_NULL_TERMINATED,
+        WOLFSENTRY_ACTION_FLAG_NONE,
+        test_action,
+        NULL,
+        &id);
+
+    wolfsentry_action_insert(
+        wolfsentry,
+        "handle-connect2",
+        WOLFSENTRY_LENGTH_NULL_TERMINATED,
+        WOLFSENTRY_ACTION_FLAG_NONE,
+        test_action,
+        NULL,
+        &id);
+
+
+    char err_buf[512];
+    struct wolfsentry_json_process_state *jps;
+
+    /* Initalize the wolfSentry JSON parser */
+    if ((ret = wolfsentry_config_json_init(
+                   wolfsentry,
+                   WOLFSENTRY_CONFIG_LOAD_FLAG_NONE,
+                   &jps)) < 0)
+    {
+        printf("error while initializing wolfSentry config parser, code: %ld\n", WOLFSENTRY_ERROR_DECODE_ERROR_CODE(ret));
```

Comment:
same story, opportunity to use `WOLFSENTRY_ERROR_FMT` and `WOLFSENTRY_ERROR_FMT_ARGS`.